### DISCUSSION TITLE
Rowy build fails

### DISCUSSION THREAD

**Discussion by RajGM** [AntlerVC/firetable#997] [Support / Q&A] [answered]
I have enabled the billing, and the rowy run is set up, but the rowy build fails each time.

Stderr: error Command failed with exit code 2

**Comment by bofeiw** [comment, marked as answer]
@RajGM Can you share more information? Do you have screenshots and code that failed to build?

**Reply by BeeBombshell** [reply]
This issue was resolved in the recent [office hours](https://discord.gg/vZbFQzje?event=1059820599184195604) meeting on Discord. Therefore, closing this discussion.
